lib/kitchen/configurable.rb

```diff
@@ -390,8 +390,8 @@ def plugin_version(version) # rubocop:disable Style/TrivialAccessors
       def diagnose
         {
           class: name,
-          version: @plugin_version,
-          api_version: @api_version,
+          version: @plugin_version ||= nil,
```

Comment:
They get used other places. They aren't initialized in a traditional manner (i.e. in an `initialize()` method) and usage of them in some places makes a modern Ruby interpreter bark about uninitialized variables. I could not determine a good place to initialize them because they're supposed to be set specific to a kitchen plugin via the DSL.